staging/src/k8s.io/cli-runtime/pkg/resource/visitor.go

```diff
@@ -425,18 +427,25 @@ func (v FlattenListVisitor) Visit(fn VisitorFunc) error {
 		if info.Mapping != nil && !info.Mapping.GroupVersionKind.Empty() {
 			preferredGVKs = append(preferredGVKs, info.Mapping.GroupVersionKind)
 		}
-
+		errs := []error{}
 		for i := range items {
 			item, err := v.mapper.infoForObject(items[i], v.typer, preferredGVKs)
```

Comment:
based on the CI failure it looks like we might also need a check for if `item` is `nil` (eg, in the case of a 404) before the call on line 439 https://github.com/kubernetes/kubernetes/pull/81452/files#diff-137886e1227323647f2d617413a7f0a0R439

Reply:
Or maybe not that specific line, but at the top of `Visit`. Either way I think it will need to check for nil since now we don't just return on errors after calling `infoForObject`